release_notes/ocp-4-18-release-notes.adoc

```diff
@@ -112,11 +115,69 @@ Alternatively, automatic z-stream updates provide a seamless and secure experien
+
+The updated location and guide name reflect a more focused documentation experience and aims to differentiate between {olmv1} and {olmv0}.
+====
+
 [id="ocp-4-18-extensions-supported-extensions_{context}"]
-==== {olmv1} supported extensions and known issue
+==== {olmv1} supported extensions
 
 include::snippets/olmv1-tp-extension-support.adoc[]
 
+[id="ocp-4-18-extensions-disconnected_{context}"]
+==== Disconnected environment support in {olmv1}
+
+To support cluster administrators that prioritize high security by running their clusters in internet-disconnected environments, especially for mission-critical production workloads, {olmv1} includes cluster extension lifecycle management functionality that works within these disconnected environments, starting in {product-title} {product-version}.
+
+After using the oc-mirror plugin for the {oc-first} to mirror the images required for your cluster to a mirror registry in your fully or partially disconnected environments, {olmv1} can function properly in these environments by utilizing the sets of resources generated by either oc-mirror plugin v1 or v2.
+
+For more information, see xref:../extensions/catalogs/disconnected-catalogs.adoc#disconnected-catalogs[Disconnected environment support in {olmv1}].
+
+[id="ocp-4-18-extensions-catalog-selection_{context}"]
+==== Improved catalog selection in {olmv1}
+
+With this release, you can perform the following actions to control the selection of catalog content when you install or update a cluster extension:
+
+* Specify labels to select the catalog
+* Use match expressions to filter across catalogs
+* Set catalog priority
+
+For more information, see xref:../extensions/catalogs/catalog-content-resolution.adoc#catalog-content-resolution[Catalog content resolution].
+
+[id="ocp-4-18-extensions-proxy-and-trustedCA_{context}"]
+==== Basic support for proxied environments and trusted CA certificates
+
+With this release, Operator Controller and catalogd can now run in proxied environments and include basic support for trusted CA certificates.
+
+[id="ocp-4-18-extensions-maxopenshiftversion_{context}"]
+==== Compatibility with {product-title} versions
+
+Before cluster administrators can update their {product-title} cluster to its next minor version, they must ensure that all installed Operators are updated to a bundle version that is compatible with the cluster's next minor version (4.y+1).
+
+Starting in {product-title} {product-version}, {olmv1} supports the `olm.maxOpenShiftVersion` annotation in an Operator's cluster service version (CSV), similar to the behavior in {olmv0}, to prevent administrators from updating the cluster before updating the installed Operator to a compatible version.
```

Comment:
```suggestion
Starting in {product-title} {product-version}, {olmv1} supports the `olm.maxOpenShiftVersion` annotation in a cluster service version (CSV) of an Operator, similar to the behavior in {olmv0}, to prevent administrators from updating the cluster before updating the installed Operator to a compatible version.
```

Same comment